tools/gen_ort_dockerfile.py

```diff
@@ -32,6 +32,13 @@
 
 FLAGS = None
 
+ORT_TO_TRTPARSER_VERSION_MAP = {
+    '1.9.1': (
+        '8.2',         # TensorRT version
```

Comment:
The TensorRT versions that you will see will be a.b.c.d or even something like 8.0.3.4+cuda11.3.1.005
So I think you will need to parse --trt-version to look only at a.b portion of the version